images/toolbox/toolbox.sh

```diff
@@ -49,8 +46,8 @@ export ROOK_API_SERVER_ENDPOINT=${ROOK_API_SERVER_ENDPOINT}
 
 EOF
 
-    CLIENT_INFO=$(curl -s http://${ROOK_API_SERVER_ENDPOINT}/client)
-    MON_ENDPOINTS=$(echo $CLIENT_INFO | jq -r '.monAddresses[]' | awk -F '/' 'BEGIN { ORS="," }; {print $1}' | sed 's/,$//')
+    shopt -s extglob
```

Comment:
this is just the first command we found yesterday that does what we want, but there may be a more proper (idiomatic) way to use extended globbing from within a script (as opposed to an interactive terminal).  also, does this shell option need to be turned back off after we've executed the command on the next line?